extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -492,13 +492,15 @@ export class InstrumentationService {
   createTimerListener_(listener, timerSpec) {
     const hasImmediate = timerSpec.hasOwnProperty('immediate');
     const callImmediate = hasImmediate ? Boolean(timerSpec.immediate) : true;
-    const intervalId = this.win_.setInterval(
-      listener.bind(null, new AnalyticsEvent(AnalyticsEventType.TIMER)),
-      timerSpec['interval'] * 1000
-    );
+    const viewer = this.viewer_;
+    const intervalId = this.win_.setInterval(() => {
```

Comment:
Let's hoist this function instead of passing it. Then we can call it in the `callImmediate` conditional below.